index.css

```diff
@@ -0,0 +1,143 @@
+.mobile {
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  background-image: url(images/header-shapes.png);
+  display: flex;
+  flex-direction: column;
+}
+
+.navbar {
+  position: absolute;
+  width: 375px;
+  height: 92px;
+}
+
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  top: 44px;
+  align-items: center;
+}
+
+img {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-end;
+}
+
+.toggle {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  position: absolute;
+  width: 18px;
+  height: 12px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  margin-top: 35px;
+  margin-right: 24px;
+}
+
+.logo {
+  width: 76px;
+  height: 20px;
+  font-family: 'Courier New', Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  margin-top: 58px;
+  margin-left: 24px;
+}
+
+.headerline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  width: 375px;
+  height: 604px;
+  left: calc(50% - 375px / 2);
+  top: 92px;
+  position: absolute;
+  background-image: url(images/header-shapes.png);
+}
+
+.headline h2 {
+  width: 327px;
+  height: 150px;
+  margin-top: 114px;
+  margin-left: 24px;
+  margin-right: 24px;
+  font-family: 'Courier New', Courier, monospace;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.headline p {
+  width: 327px;
+  height: 146px;
+  font-family: 'Courier New', Courier, monospace;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+  top: 368px;
+  padding-top: 12px;
+}
```

Comment:
- Kindly add the  `margin-left: 24px;`  so it will move a bit the paragraph and it will put your body container in the same line as it is shown in the Figma design provided. check the difference below between your design and the Figma design :grinning::grinning: :grinning::grinning:

- Kindly ensure that the font family, font size, line height, and font weight of the text in your body stick to the Figma design you chose as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). You can check out the Figma design specification for the exact properties used👍🏽 
- I suggest importing this font family I show you below  so that you can apply it in your all  body :+1: :grinning: :grinning: :grinning: 

```
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");

 font-family: "Poppins", sans-serif;

```
## Your design :-1: 

![image](https://user-images.githubusercontent.com/95297251/191970517-4604bb72-4b18-4625-aa13-c686ca025078.png)


## Figma design :+1: 

![image](https://user-images.githubusercontent.com/95297251/191950195-ecd552d1-2ed7-4c5c-b94c-6f5a445800d9.png)